artemis-prometheus-metrics-plugin/src/main/java/com/redhat/amq/broker/core/server/metrics/plugins/ArtemisPrometheusMetricsPlugin.java

```diff
@@ -26,11 +26,11 @@
 
 public class ArtemisPrometheusMetricsPlugin implements ActiveMQMetricsPlugin {
 
-   MeterRegistry meterRegistry;
+   private MeterRegistry meterRegistry;
 
    @Override
-   public ActiveMQMetricsPlugin init(Map<String, String> options) {
-      this.meterRegistry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
+   public ActiveMQMetricsPlugin init(final Map<String, String> options) {
```

Comment:
Is this change required?

Reply:
No, not required, but making a variable final if it should not change is my preference. Should I revert it?

Reply:
I see your point but I usually avoid changes not required or not related to the commit subject to avoid confusion for future contributors that need to review the commit